opm/test_util/summaryRegressionTest.cpp

```diff
@@ -60,7 +62,7 @@ void RegressionTest::getRegressionTest(){
                 if (isRestartFile && keywordString.substr(3,1)=="T"){
                     break;
                 }
-                checkForKeyword(timeVec1, timeVec2, keyword);
+                throwAtEnd = !checkForKeyword(timeVec1, timeVec2, keyword);
```

Comment:
Shouldn't this be something like `throwAtEnd |= !checkForKeyword( );` to carry a `true` value through the keyword list?